packaging/examples/Makefile

```diff
@@ -6,6 +6,7 @@ RELEASE_VERSION ?= latest
 RELEASE_PATH ?= ../../strimzi-$(RELEASE_VERSION)/$(PROJECT_NAME)
 
 release:
+	$(SED) -Ei -e "s#strimzi.io/docs/operators/latest/using.html#strimzi.io/docs/operators/$(RELEASE_VERSION)/using.html#" security/keycloak-authorization/README.md
```

Comment:
I do not think this will work. The releases go through several stages. It first changes. For example ...
* the original link in master should be `strimzi.io/docs/operators/in-development/using.html`
* For 0.23.0-rc1 it should changed from `strimzi.io/docs/operators/in-development/using.html` to `strimzi.io/docs/operators/0.23.0-rc1/using.html`
* Then for final 0.23.0 release from `strimzi.io/docs/operators/0.23.0-rc1/using.html` to `strimzi.io/docs/operators/0.23.0/using.html`
* And for patch release from `strimzi.io/docs/operators/0.23.0/using.html` to `strimzi.io/docs/operators/0.23.1/using.html`
* etc.

So the replacement needs to use some pattern / wildcard in the source.

Reply:
@scholzj  I see what you mean. At least two changes are needed:
- Link in the document should be: strimzi.io/docs/operators/in-development/using.html
- Regex should be: "s#strimzi.io/docs/operators/[^/]+/using.html#strimzi.io/docs/operators/$(RELEASE_VERSION)/using.html#"

But then, is 'in-development' a RELEASE_VERSION or is it something else? The regex uses RELEASE_VERSION for replacement, and that one defaults to `latest`, thus the output of `make` will never preserve `in-development`, even for regular build. Is that the desired behavior, or should we introduce DOC_TAG variable that would preserve `in-development` for regular builds?

Reply:
Looks like DOC_TAG is needed.